packages/caliper-core/lib/worker/rate-control/recordRate.js

```diff
@@ -89,7 +89,10 @@ class RecordRateController extends RateInterface {
      */
     _exportToText() {
         fs.writeFileSync(this.pathTemplate, '', 'utf-8');
-        this.records.forEach(submitTime => fs.appendFileSync(this.pathTemplate, `${submitTime}\n`));
+        this.records.forEach(submitTime => {
```

Comment:
Oh okay, would look at the docs to fix the bug

Reply:
@tunedev it's something that's been in the code for a while by the looks of things and it works when only one request is sent in the sendRequests call, but now thanks to you putting tests in which would simulate multiple requests in a sendRequests call, it would fail to do the right thing.